lib/utilities/monitoringHandler.js

```diff
@@ -3,65 +3,65 @@ const client = require('prom-client');
 const numberOfBuckets = new client.Gauge({
-    name: 'cloud_server_number_of_buckets',
+    name: 's3_cloudserver_buckets',
     help: 'Total number of buckets',
 });
 const numberOfObjects = new client.Gauge({
-    name: 'cloud_server_number_of_objects',
+    name: 's3_cloudserver_objects',
     help: 'Total number of objects',
 });
 const lastReportTimestamp = new client.Gauge({
-    name: 'cloud_server_last_report_timestamp',
+    name: 's3_cloudserver_last_report_timestamp',
     help: 'Timestamp of last object/bucket count report',
 });
 const numberOfIngestedObjects = new client.Gauge({
-    name: 'cloud_server_number_of_ingested_objects',
-    help: 'Number of out of band ingestion',
+    name: 's3_cloudserver_ingested_objects',
+    help: 'Total number of out of band objects ingested',
 });
 const dataIngested = new client.Gauge({
-    name: 'cloud_server_data_ingested',
+    name: 's3_cloudserver_ingested_bytes',
     help: 'Cumulative size of data ingested in bytes',
 });
 const dataDiskAvailable = new client.Gauge({
-    name: 'cloud_server_data_disk_available',
+    name: 's3_cloudserver_disk_available_bytes',
     help: 'Available data disk storage in bytes',
 });
 const dataDiskFree = new client.Gauge({
-    name: 'cloud_server_data_disk_free',
+    name: 's3_cloudserver_disk_free_bytes',
     help: 'Free data disk storage in bytes',
 });
 const dataDiskTotal = new client.Gauge({
-    name: 'cloud_server_data_disk_total',
+    name: 's3_cloudserver_disk_bytes',
     help: 'Total data disk storage in bytes',
 });
 
 const labelNames = ['method', 'action', 'code'];
 const httpRequestsTotal = new client.Counter({
     labelNames,
-    name: 'http_requests_total',
-    help: 'Total number of HTTP requests',
+    name: 's3_cloudserver_http_requests_total',
+    help: 'Total number of cloudserver HTTP requests',
 });
 const httpRequestDurationSeconds = new client.Histogram({
-    name: 'http_request_duration_seconds',
+    name: 's3_cloudserver_http_request_duration_seconds',
     help: 'Duration of HTTP requests in seconds',
     labelNames,
     buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5], // buckets for response time from 0.1ms to 500ms
 });
 const httpActiveRequests = new client.Gauge({
-    name: 'http_active_requests',
-    help: 'Number of HTTP in-flight requests',
+    name: 's3_cloudserver_http_active_requests',
+    help: 'Number of cloudserver HTTP in-flight requests',
 });
 
 const httpRequestSizeBytes = new client.Summary({
     labelNames,
-    name: 'http_request_size_bytes',
-    help: 'The HTTP request sizes in bytes.',
+    name: 's3_cloudserver_http_request_size_bytes',
+    help: 'Cloudserver HTTP request sizes in bytes',
 });
 
 const httpResponseSizeBytes = new client.Summary({
     labelNames,
-    name: 'http_response_size_bytes',
-    help: 'The HTTP response sizes in bytes.',
+    name: 's3_cloudserver_http_response_size_bytes',
+    help: 'Cloudserver HTTP response sizes in bytes',
```

Comment:
This is a standard metrics, that is why a standard name is used.

Reply:
same as above